### ISSUE TITLE
Make focus states visible on mobile/tablet filter panel

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6471]
On smaller resolutions, the filter panel on the search page is presented on a blue background:

![Screenshot from 2019-08-17 06-55-50](https://user-images.githubusercontent.com/3323703/63207356-57be8880-c0bc-11e9-97cd-7075623b53a5.png)

When this happens, the focus states for the accordion component and the checkboxes is not visible.

We need to update the code so that we have a white focus state for these elements, when they are presented on the blue bg.
